pathspider/chains/trace_ecn.py

```diff
@@ -0,0 +1,97 @@
+"""
+.. module:: pathspider.chains.traceroute
+   :synopsis: A flow analysis chain for traceroute messages especially icmp messages
+
+"""
+from pathspider.chains.base import Chain
+
+class ECNChain_trace(Chain):
```

Comment:
I wonder if traceroute chains are sufficiently different from regular chains that they should be a seperate category? If they complement the existing chains then the traceroute specific functions could just be added into the existing chains. Having traceroute specific chains but still calling them chains feels wrong to me.